qemu/tests/cfg/block_hotplug.cfg

```diff
@@ -59,6 +59,42 @@
             sub_type_after_plug = boot
             sub_type_after_unplug = boot
             reboot_method = system_reset
+        - with_block_resize:
+            only with_plug
+            sub_type_after_plug = block_resize
+            blk_extra_params_stg0 += " serial=TARGET_DISK0"
+            block_size_cmd = "fdisk -l | grep {0}"
+            block_size_pattern = ",\s+(\d+\s+b)ytes"
+            disk_change_ratio = "1.5 0.5"
+            repeat_times = 1
+            Windows:
+                block_size_cmd = "wmic diskdrive get size, index"
+                block_size_pattern = "1\s+(\d+)"
+                accept_ratio = 0.0005
+                disk_update_cmd = "echo rescan > cmd"
+                disk_update_cmd = "echo select disk 1 > cmd"
```

Comment:
Is it should be `disk_update_cmd+ = && "echo select disk 1 > cmd"`?

Reply:
yes, you are right, will update it, thank you.